coderefs/coderefs.go

```diff
@@ -154,30 +168,31 @@ func runExtinctions(opts options.Options, matcher search.Matcher, branch ld.Bran
 	lookback := opts.Lookback
 	dryRun := opts.DryRun
 	if lookback > 0 {
-		missingFlags := []string{}
-		for flag, count := range branch.CountByFlag(matcher.Elements[0].Elements) {
-			if count == 0 {
-				missingFlags = append(missingFlags, flag)
-			}
+		var removedFlags []ld.ExtinctionRep
+		for i, project := range opts.Projects {
```

Comment:
Loop over all of the projects now to find extinctions. The API endpoint already accepts an array of extinctions and each extinction has its project key embedded in the struct.